wiki/Beatmapping/Timing/en.md

```diff
@@ -0,0 +1,55 @@
+
+## Song structure
+
+*Main article: [Music theory](/wiki/Music_theory)*
+
+Timing is based on a fact that most songs adhere to certain rules and guidelines and have rhythm — a regularly repeated sequence of sounds and silences:
+
+- [Beat](/wiki/Music_theory/Beat) is a note considered a basic unit of rhythm. Other notes are expressed as its fractions and multiples.
+- [Measure](/wiki/Music_theory/Measure), or a bar, is a fixed amount of beats repeating throughout the song. The first beat of a measure is emphasized and called a [downbeat](/wiki/Music_theory/Downbeat).
+- [Time signature](/wiki/Music_theory/Time_signature) specifies how many beats are in a single measure.
+- [BPM](/wiki/Music_theory/Tempo), or beats per minute, is a value that determines the speed of a song.
+
+## Timing process
+
+*Main article: [How to time songs](/wiki/Guides/How_to_Time_Songs)*\
+*See also: [Beatmap editor — Timing](/wiki/Client/Beatmap_editor/Timing)*
+
+### Single offset
+
+An [uninherited ("red") offset](/wiki/Beatmapping/Offset) shows the amount of time since the beginning of the audio before the first downbeat. The offset is a starting point used by osu! to calculate positions of other beats. A single offset is often enough for songs that are digitally produced or edited seamlessly.
+
+Mappers use their perception and sense of rhythm to discover the basic rhythmic structure of a song, and then verify that it accurately represents the whole audio. To do that, they focus on similar sounds which repeat periodically, and tap the `T` key along the rhythm. This allows osu! to calculate how many beats appear in a song over a minute, and find its approximate [BPM](/wiki/Beatmapping/Beats_per_minute). After that, the timing is tested and adjusted if necessary:
+
+- BPM may be rounded if it yields more precise results;
+- The offset [needs to be moved](/wiki/Guides/Setting_the_Offset_on_the_Correct_Beat) to the first downbeat of a song.
+
+### Multiple offsets
+
+![](img/complex-timing.png "Example timeline of a beatmap with complex timing, \"MiddleIsland — Roze\" by Lan wings. Nearly 300 uninherited offsets describe 3 minutes of piano play.")
+
+A beatmap can also have several offsets, provided they accurately represent the underlying song.
+
+Most often, a song has sections with different amount of beats per minute. The change may be noticeable and expressed clearly, in which case an extra timing point is usually enough. On the other hand, gradual tempo changes require the mapper to slowly adjust the timing, adding dozens and sometimes hundreds of new offsets.
+
+If the rhythm isn't ideal, new timing points may be added to account for discrepancies between expected and real notes. These artefacts are often caused by imprecise audio edits, or by music performed without metronome. As a result, the final composition may have arbitrary tempo changes and random note drifts.
```

Comment:
worth mentioning that this and the above example are difficult to get correct, so it's better to avoid these songs without experience